nexus/peer-snowflake/src/auth.rs

```diff
@@ -6,9 +6,9 @@ use std::{
 use anyhow::Context;
 use base64::prelude::{Engine as _, BASE64_STANDARD};
 use jsonwebtoken::{encode as jwt_encode, Algorithm, EncodingKey, Header};
-use pkcs1::EncodeRsaPrivateKey;
-use pkcs8::{DecodePrivateKey, EncodePublicKey};
 use rsa::{RsaPrivateKey, RsaPublicKey};
+use rsa::pkcs1::EncodeRsaPrivateKey;
```

Comment:
Seems like this isn't used

Reply:
Trait needs to be imported to call trait methods